src/file-manager/fm-desktop-icon-view.c

```diff
@@ -38,6 +38,9 @@
 #include <gdk/gdkx.h>
 #include <glib/gi18n.h>
 #include <libcaja-private/caja-desktop-icon-file.h>
+#if !GTK_CHECK_VERSION(3, 21, 0)
```

Comment:
Looks like this isn't needed, that header is already included a few lines below :smile: